RegExp,Date.js

```diff
@@ -0,0 +1,18 @@
+/*Дана строка 'ahb acb aeb aeeb adcb axeb'. Напишите регулярное выражение, 
+которое найдет строки ahb, acb, aeb по шаблону: буква 'a', любой символ, буква 'b'*/
+let str = 'ahb acb aeb aeeb adcb axeb';
+let regexp = /a.b/g;
+let result = str.match(regexp);
+console.log(result);
+
+//Дана строка '2+3 223 2223'. Напишите регулярку, которая найдет строку 2+3, не захватив остальные
+let str2 = '2+3 223 2223';
+let regexp2 = /\d\D\d\s/;
```

Comment:
Not fully appropriate regexp. It is not working for string '2+3 223 2223 2+3 asdasd21321 134 123 124 124 41'. Please fix.